lib/uefi/configuration_table.cpp

```diff
@@ -17,15 +17,24 @@
 
 #include "configuration_table.h"
 #include "boot_service_provider.h"
+#include "libfdt.h"
+#include "platform.h"
 #include "system_table.h"
 #include <string.h>
 
 void setup_configuration_table(EfiSystemTable *table) {
+  const void *fdt = get_fdt();
```

Comment:
Would be pretty wise to null check this.

Reply:
Done